apps/next-product-starter/src/pages/video/[slug].tsx

```diff
@@ -253,13 +267,15 @@ const Page = ({data}: any) => {
         }}
         actions={{
           onVideoEnded: (_context: any, _event: any) => {
-            console.log('do stuff')
+            if (getPlayerPrefs().autoplay && !isEmpty(nextLesson)) {
+              router.push(`/video/${nextLesson.slug}`)
+            }
```

Comment:
```suggestion
            const {autoplay} = getPlayerPrefs()
            const hasNextLesson = !isEmpty(nextLesson)
            if (autoplay && hasNextLesson) {
              router.push(`/video/${nextLesson.slug}`)
            }
```

suggestion (style): I'm generally a big fan of taking an explicit approach so that the logic reads